scripts/webpack/webpack-resources.js

```diff
@@ -350,15 +349,17 @@ module.exports = {
         // Use the aliases for react-examples since the examples and demo may depend on some things
         // that the package itself doesn't (and it will include the aliases for all the package's deps)
         alias: getResolveAlias(false /*useLib*/, reactExamples),
+        fallback: {
+          path: require.resolve('path-browserify'),
```

Comment:
What does this do?

Reply:
https://stackoverflow.com/questions/64557638/how-to-polyfill-node-core-modules-in-webpack-5
https://github.com/browserify/path-browserify

replaces references with polyfill, since webpack no longer does this for you.